tests/test_TFNetworkLayer.py

```diff
@@ -150,6 +150,141 @@ def test_concat_sources_missing_dim():
     session.run(out.output.placeholder)
 
 
+def test_self_att_config():
```

Comment:
Please use more meaningful names. The function name should point out what issue you are testing here (e.g. `test_self_att_time_dim_tags` or so; later when you understood what the problem is, you can maybe even use a better name).

Reply:
Done